reps_event_app/lib/ui/events.dart

```diff
@@ -105,7 +105,6 @@ class _EventsState extends State<Events> {
                     scrollDirection: Axis.vertical,
                     itemCount: snapshot.data.length,
                     itemBuilder: (context, index) {
-                      ;
```

Comment:
I don't think there should be a semi-colon here. Have you tested this on your device/emulator?

Reply:
I have removed it